### ISSUE TITLE
Cypress 10

### ISSUE THREAD

**Issue by reintroducing** [adam-lynch/preprocess-cypress-tests-with-vite#2]
Just wanted to follow up here that I don't think this is required anymore with Cypress 10. I seem to be getting everything working without the need for this, but wanted to see if I'm missing something or if you agree that this is "no longer a thing".

**Comment by reintroducing** [comment]
I stand corrected, this is still needed for e2e tests but not for component tests. e2e tests WILL work just fine without it, HOWEVER, in my case, I'm using msw for mocking. in my msw setup i have a mocks directory that has all my handlers/responses split out by directory structure that mirrors the api paths. i then have an index.js file that loads all those mocks automatically into the app and/or test framework (since I use mocks in both dev and testing). That file accomplishes this by doing the following:
```js
const req = import.meta.globEager('./**/index.js');
const modules = [];

Object.keys(req).forEach(key => {
    modules.push(...req[key].default);
});

export default modules;
```

You can see that i'm using `import.meta.globEager` which is "a Vite thing" and as such, without processing the e2e tests through this setup, it fails because it thinks it needs another loader to process the `import.meta` statements.

In short, if you aren't doing anything Vite specific in your files, you will not need this setup. If you do, as I am above, you will.

Closing this as I've answered my own question :)

**Comment by adam-lynch** [comment]
Thanks 👍
